openpype/hosts/blender/plugins/load/load_abc.py

```diff
@@ -131,7 +130,20 @@ def process_asset(
 
         objects = self._process(libpath, asset_group, group_name)
 
-        bpy.context.scene.collection.objects.link(asset_group)
+        # Link the asset group to the active collection
+        collection = bpy.context.view_layer.active_layer_collection.collection
+        collection.objects.link(asset_group)
+
+        # Link the imported objects to any collection where the asset group is
+        # linked to, except the AVALON_CONTAINERS collection
+        group_collections = [
+            collection
+            for collection in asset_group.users_collection
+            if collection != avalon_containers]
+
+        for obj in objects:
+            for collection in group_collections:
+                collection.objects.link(obj)
```

Comment:
maybe we can put these line of codes(136-146)  as a function in lib and implement that function here knowing that these lines are similar to what has been implemented in line 227-234?